custom_model_runner/datarobot_drum/resource/components/Python/generic_predictor/generic_predictor.py

```diff
@@ -105,7 +106,13 @@ def _materialize(self, parent_data_objs, user_data):
             binary_data, mimetype = StructuredInputReadUtils.read_structured_input_file_as_binary(
                 input_filename
             )
-            transformed_df = self._predictor.transform(binary_data=binary_data, mimetype=mimetype)
+            transformed_output = self._predictor.transform(
+                binary_data=binary_data, mimetype=mimetype
+            )
+            if type(transformed_output) == tuple:
+                transformed_df = transformed_output[0]
+            elif type(transformed_output) == DataFrame:
+                transfromed_df = transformed_output
```

Comment:
```suggestion
            if isinstance(transformed_output, tuple):
                transformed_df = transformed_output[0]
            elif isinstance(transformed_output, DataFrame):
                transfromed_df = transformed_output
```

Reply:
changed.  

need to figure out what to test here.  @yakov-g any thoughts?

Reply:
The test could be to check cases when transformed_output is either tuple or DataFrame, but, as I said before, looks like PythonPredictor always returns tuple. And transform is not implemented for other languages. 

So I cant think of any new test cases.

Reply:
resolve -> remove checks.  I agree that the PythonPredictor always returns a tuple.